File: Problem Solving/Python/primelist.py
#Finding the sum of prime numbers upto nth number
print("Enter the range value")
r=int(input())
sum=0
print("Prime number sequence")
for i in range(1,r+1):
    count=0
    for j in range(1,i+1):
        if(i%j==0):
            count=count+1
    if(count==2):
        print(i)
        sum=sum+i
print(sum)

File: Problem Solving/C/primelist.c
/*print prime number upto a given sequence*/
#include<stdio.h>
#include<conio.h>
void main()
{
int num;
printf("enter the range upto you want prime sequence");
scanf("%d",&num);
for(int i=2;i<=num;i++)
{
int count=0;
for(int j=1;j<=i;j++)
{
if(i%j==0)
{
count++;
}
}
if(count==2)
{
printf("%d\n",i);
}
}
getch();
}


